lib/full_node.js

```diff
@@ -0,0 +1,14 @@
+//      
+var bcoin = require('bcoin');
+var TrustIsRisk = require('./trust_is_risk');
+
+class FullNode extends bcoin.fullnode {
+                     
+
+  constructor(options         ) {
+    super(options);
+    this.trust = new TrustIsRisk(this);
```

Comment:
Trust is risk requires a full node instance? Can you discuss this architectural decision? It seems that the TIR library may need less than that.